src/main/java/com/juniorstart/juniorstart/controller/UserController.java

```diff
@@ -58,4 +59,16 @@ public User getCurrentUser(@CurrentUser UserPrincipal userPrincipal) {
     public ResponseEntity<?> changePassword(@RequestBody @Valid ChangePasswordRequest changePassword){
         return userService.changePassword(changePassword);
     }
+
+    @PostMapping("changeStatus")
+    @PreAuthorize("hasRole('USER')")
+    public ResponseEntity<?> changeStatus(@RequestBody @Valid ChangeStatusRequest request){
+        return userService.changeStatus(request);
+    }
+
+    @GetMapping("statusList")
+    @PreAuthorize("hasRole('USER')")
+    public ResponseEntity<?> getStatusList(){
```

Comment:
Za co to odpowiada? 🤔

Reply:
Aby człowiek od front endu dostawał od razu liste statusów jakie ma umieścić na stronie. Jak dodamy jakiś albo zmienimy to będzie mieć od razu.

Reply:
A, no dobra